examples/00-load/create-point-cloud.py

```diff
@@ -90,7 +91,7 @@ def generate_points(subset=0.02):
 # This time, we're going to create a totally new, random point cloud.
 
 # Create random XYZ points
-points = np.random.default_rng().random((100, 3))
+points = np.random.default_rng(seed=1).random((100, 3))
```

Comment:
I personally think it would be more clear if all examples and docstrings that need random numbers follow the same structure, i.e. importing the required packages, seeding an `rng` object, and continuing with the actual demo code (as is done in most other places in this PR). So no single line `rng` creation and usage as is done here. But that's just my 2 cents :)

Reply:
This is also the recommended by scientific Python:

https://blog.scientific-python.org/numpy/numpy-rng/

"You should create an RNG at the beginning of your script (with your own seed if you want reproducibility) and use this RNG in the rest of the script."

E.g.

``` python 
import numpy as np

rng = np.random.default_rng()
rng.random()  # generate a floating point number between 0 and 1
```